src/components/SavedNews/SavedNews.css

```diff
@@ -0,0 +1,86 @@
+.saved__news-header {
+  display: flex;
+  flex-direction: column;
+  padding: 40px 24px 56px;
+  margin: 0 auto;
+  max-width: 1280px;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.saved__news--page {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 18px;
+  color: #1a1b2280;
+  width: 100%;
+}
+.saved__news--title {
+  margin: 0;
+  padding: 28px 0 30px;
+  max-width: 530px;
+  font-family: "Roboto Slab", Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 46px;
+  color: #1a1b22;
+}
+
+.saved__keywords {
+  margin: 0;
+  padding: 0;
+  max-width: 712px;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 24px;
+  color: #1a1b22;
+}
+.saved__keywords-text {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  columns: #1a1b22;
```

Comment:
`  columns: #1a1b22;
`

Invalid `css` property value. Please, check syntax. 

It should be `color`